File: A2Z_sheet/Array/Easy/02_second_largest_in_array/solution.java
class Solution {
    public int print2largest(List<Integer> arr) {
        int max = -1, secMax = -1;
        for(int num: arr){
            if(num > max){
                secMax = max;
                max = num;
            }
            else if(num > secMax && num < max) secMax = num;
        }
        return secMax;
    }
}

File: A2Z_sheet/Array/Easy/02_second_largest_in_array/solution.js
class Solution {
    print2largest(arr) {
        let [max, secMax] = [-1, -1];
        for (const num of arr) {
            if (num > max) [secMax, max] = [max, num];
            else if (num > secMax && num < max) secMax = num;
        }
        return secMax;
    }
}
